### PR TITLE
Expose account name and FQDN

### PR DESCRIPTION
**Pull request by johan3141592**

### CODE REVIEW

File: pkg/polaris/config.go [hunk 1]
```diff
@@ -24,95 +24,85 @@ import (
...
+	// Optional RSC API endpoint. Useful for running the SDK against a test
+	// service. When omitted, it defaults to https://{Name}.my.rubrik.com/api.
 	URL string
 
+	accountName string
+	accountFQDN string
+	apiURL      string
 	envOverride bool
+	tokenURL    string
 }
```
Comment (Draper1) [inline pkg/polaris/config.go:54]: nit: could we annotate these new struct values?

File: pkg/polaris/config.go [hunk 2]
```diff
@@ -268,144 +224,217 @@ func UserAccountFromFile(file, name string, allowEnvOverride bool) (*UserAccount
...
+	ClientID       string `json:"client_id"`        // Client ID.
+	ClientSecret   string `json:"client_secret"`    // Client secret.
+	Name           string `json:"name"`             // Service account name.
+	AccessTokenURI string `json:"access_token_uri"` // Access token URI.
+
+	accountName string
+	accountFQDN string
+	apiURL      string
+	envOverride bool
+	tokenURL    string
```
Comment (Draper1) [inline pkg/polaris/config.go:365]: nit: comments for additional struct values

**Review by Draper1 — APPROVED** [review]

File: pkg/polaris/config.go (hunk 1 above)
Comment (johan3141592) [reply pkg/polaris/config.go:54]: We primarily document the public API, in addition to this the `ServiceAccount`/`UserAccount` struct has additional documentation for the public access functions for each of the fields.

File: pkg/polaris/config.go (hunk 2 above)
Comment (johan3141592) [reply pkg/polaris/config.go:365]: We primarily document the public API, in addition to this the `ServiceAccount`/`UserAccount` struct has additional documentation for the public access functions for each of the fields.

**Pull request merged**